### PR TITLE
[CFM-1204] Adding my groups menu on the top bar.

### PR DESCRIPTION
**Pull request by ordavidil**
https://issuetracker.amplexor.com/jira/browse/CFM-1204

### CODE REVIEW

File: project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_users.scss [hunk 1]
```diff
@@ -130,3 +134,7 @@
 .field-name-c4m-job-title {
   margin-bottom: 0.5rem;
 }
+
+.user-groups {
```
Comment (ordavidil) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_users.scss]: @guytepper 
this is too generic, probably should be under `#user-bar`

Comment (guytepper) [inline project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_users.scss]: I know, it's temporary.

**Comment by guytepper** [comment]
Mobile:
screen shot 2016-08-15 at 3 52 53 pm

Tablet:
screen shot 2016-08-15 at 3 53 06 pm

Desktop:
screen shot 2016-08-15 at 3 53 21 pm

**Comment by guytepper** [comment]
**Mobile:**
screen shot 2016-08-15 at 3 52 53 pm

**Tablet:**
screen shot 2016-08-15 at 3 53 06 pm

**Desktop:**
screen shot 2016-08-15 at 3 53 21 pm

**Comment by ordavidil** [comment]
Removed the bottom border of the last item:
selection_258

**Comment by ordavidil** [comment]
@soniCaH 
Updated the logic
selection_259

**Comment by soniCaH** [comment]
@ordavidil / @guytepper 
could you please make the styling of the groups dropdown and the profile dropdown consistent (same border radius, same color, same height...) + right-align it as designed? 

top-bar

**Comment by guytepper** [comment]
@soniCaH

**Comment by guytepper** [comment]
@soniCaH - is that how it should look like?

## Mobile
image

## Tablet
image

## Desktop
image

**Comment by soniCaH** [comment]
@guytepper almost! :-)
The group images should have equal size, and is it possible to make it a bit wider? Even rather short group names are already split on 2 lines (believe me, there are groups with lines that will span 10 lines with this width ;-)).

**Comment by guytepper** [comment]
## Mobile (320px)
image

## Tablet (768px) 
image

## Desktop (1200px)
image

**Comment by ordavidil** [comment]
@guytepper 
on mobile it is too thin, please increase it to take 100% of the screen

**Comment by guytepper** [comment]
Mobile:
image

The other breakpoints stays the same.

**Pull request merged**